terraform/system/aws/main.tf

```diff
@@ -55,6 +55,7 @@ module "cluster" {
   key_pair_name            = var.key_pair_name
   kube_proxy_version       = var.kube_proxy_version
   k8s_version              = var.k8s_version
+  imds_http_tokens         = var.imds_http_tokens
```

Comment:
did we already has this support?

Reply:
sorry i don't see where it is being applied 
i only see these changes: https://github.com/convox/convox/pull/591/files

Reply:
yes, it's true. Some files are missing. Thank for letting me know! :)

Reply:
this is a rack parameter, so, the only change is in the tf files, right?